pkg/test/fuzz/generate.go

```diff
@@ -105,6 +105,13 @@ func fillWithRandom0(t *testing.T, randStream *rand.Rand, msg protoreflect.Messa
 					v := randomString(randStream)
 					mapVal.Set(protoreflect.ValueOf(k).MapKey(), protoreflect.ValueOf(v))
 				}
+			case "string->message":
+				if field.FullName() == "google.protobuf.Struct.fields" && field.MapValue().Message().FullName() == "google.protobuf.Value" {
+					// currently this is converted to "map[string]string" in "BigQueryDataTransferConfig"
+					// TODO: fill in random strings
```

Comment:
Not sure how to fill in random strings for this field yet. So currently `.spec.params" is not actually tested in fuzz.

Reply:
"fill in random strings" --> Do you mean the random string map? Can we do something similar to "string -> string" but replace the "v" with a new mapVal?

Reply:
I was experimenting with it but having some trouble. The fuzz test simply exit with "fuzzing process hung or terminated unexpectedly: exit status 255". I don't feel like a single field should block adding the test.